app/schemas/models/ai_chat_message.schema.js

```diff
@@ -66,6 +66,24 @@ _.extend(AIChatMessageSchema.properties, {
   },
   i18n: { title: 'I18n', type: 'object', description: 'Help translate this property', format: 'i18n', props: ['text'] },
   ownerID: { title: 'Owner Id', type: 'string', description: 'The user who created this chat message' },
+  safetyValidation: {
```

Comment:
Question - why isn't it an array instead? Isn't it possible that there are multiple types of failures for the same message? Maybe in future we'll add other types (e.g. 9yo+ or 13yo+) that should be OK for some users, but not when there is a `Violence` failure also. This implementation allows only one type, and makes the handling of above scenario much harder. Also for me the `None` type is quite strange. There _is_ a failure but with type `None`. Why is there a failure then? In case of array it would be much cleaner and understandable that failures array is empty.

Reply:
emmm good question. we can use array instead. thank you